src/ChatterServiceProvider.php

```diff
@@ -39,6 +39,17 @@ public function boot()
      */
     public function register()
     {
+        /*
+         * Register the service provider for the dependency.
+         */
+        $this->app->register('LukeTowers\Purifier\PurifierServiceProvider');
```

Comment:
Its probably the same but what happens on your end if you replace the `'LukeTowers\Purifier\PurifierServiceProvider'` with `LukeTowers\Purifier\PurifierServiceProvider::class,` it will probably ask you to import the class I guess. say yes.